CONTRIBUTING.md

```diff
@@ -43,7 +43,7 @@ Once you've forked the code and created a new branch for your work, you can run
 2. Install dependencies `poetry install`
 3. Run the program `poetry run interpreter`
 
-After modifying the source code, you will need to do `poetry run interpreter` again.
+After modifying the source code, you must do `poetry run interpreter` again.

```

Comment:
While we're making this change I wonder if we should also remove the `"do"`"?

Reply:
What about to swap out `"do"` with `"run"`, as in "run command"

> After modifying the source code, you must _run_ `poetry run interpreter` again.

Reply:
I think either `run` or removing the verb would be fine. Thanks for the suggestion @Notnaton.

@shruti222patel If you want to make one of those changes since this line is already getting changed in this PR, I'll give this one a 👍 and put it in the `reviewed` queue.

Thanks for the updates!